### ISSUE TITLE
st-util sends wrong memory map/ no features STM32F429

### ISSUE THREAD

**Issue by iabdalkader** [texane/stlink#250]
I'm having a problem with STM32F429, it seems that st-link is not sending the features packet and sends out the wrong memory map to gdb (I captured the packets with wireshark), this makes `load` fail with `flash_add_block: unaligned block` see #196 , and gdb tries to access default registers like $f0 also I was not able to access CCM.

further digging into this issue, it seems that the correct memory map and features are sent only for the `STM32_CHIPID_F4` and not `STM32_CHIPID_F4_HD` (which is what st-link reports for the STM32F429)

```
2014-07-17T00:19:02 INFO src/stlink-common.c: Device connected is: F42x and F43x device, **id 0x10036419**
```

It looks like there's no way to specify the memory map/supported packets outside the code, so a simple check for `STM32_CHIPID_F4_HD` fixes this issue, here:
https://github.com/texane/stlink/blob/master/gdbserver/gdb-server.c#L339
https://github.com/texane/stlink/blob/master/gdbserver/gdb-server.c#L739

**Comment by Nightwalker-87** [comment]
Fixed or not fixed? Please finally verify with Release 1.2.0 or any later release.

**Comment by Nightwalker-87** [comment]
@iabdalkader @landgraf: I am currently reviewing old leftovers.
Is this resolved with Release v1.2.0? Please verify.

**Comment by iabdalkader** [comment]
@Nightwalker-87 It fixed the issue for me back when this was merged (in 2014)... I don't know if this is still relevant today.

**Comment by Nightwalker-87** [comment]
Ok, waiting for @landgraf then, as he still claimed a problem afterwards.

**Comment by Nightwalker-87** [comment]
Fixed in #251.
